src/hooks/useActiveSubscription.tsx

```diff
@@ -0,0 +1,35 @@
+import { useCallback, useContext } from 'react';
+import getActiveSubscription from 'actions/getActiveSubscription';
+import useAsync from 'hooks/useAsync';
+import { TokenContext } from 'hooks/useCustomerToken';
+import { components } from '../apiTypes';
+
+type PricePlan = components['schemas']['PricePlan'];
+
+export type UseActiveSubscriptionReturnType = {
+  result: PricePlan | null;
+  loading: boolean;
+  error: unknown;
+};
+
+const useActiveSubscription = (args?: {
```

Comment:
Can you add a doc block here? 
```suggestion
/**
 * A hook that fetches the customer's active subscription information.
 */
const useActiveSubscription = (args?: {
```